tests/rptest/tests/cloud_storage_usage_test.py

```diff
@@ -123,6 +126,32 @@ def _test_epilogue(self):
         bucket_view.assert_at_least_n_uploaded_segments_compacted(
             self.topics[1].name, partition=0, revision=None, n=1)
 
+    def _check_describe_log_dirs(self):
+        quiesce_uploads(self.redpanda, [t.name for t in self.topics],
+                        timeout_sec=30)
+
+        describe_items = self.rpk.describe_log_dirs()
+        by_ntp = defaultdict(list)
+        for i in describe_items:
+            ntp = NTP(ns='kafka', topic=i.topic, partition=i.partition)
+            by_ntp[ntp].append(i)
+
+        describe_total_remote = 0
+        bucket_view = BucketView(self.redpanda)
+        for ntp, items in by_ntp.items():
+            remote_items = list(i for i in items
+                                if i.dir.startswith("remote://"))
+            self.logger.info(f"{ntp} {len(items)} describelogdirs items")
+
+            ntp_remote_size = max(i.size
+                                  for i in remote_items) if remote_items else 0
+            actual_size = bucket_view.cloud_log_size_for_ntp(
+                ntp.topic, ntp.partition)
+
+            assert ntp_remote_size == actual_size
+
+            describe_total_remote += ntp_remote_size
```

Comment:
nit: `describe_total_remote` is never used

Reply:
I think i started writing this thinking I was going to use total cloud usage to verify it, but ended up doing per-ntp -- removed this var now.